### PR TITLE
Create README_AR.rtl.md

### PR DESCRIPTION
**Pull request by 0xObsidian**
README supported in Arabic

### CODE REVIEW

File: README_AR.rtl.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+
```
Comment (RamanSB) [inline README_AR.rtl.md:1]:
```suggestion
Remove line...
```

**Review by RamanSB — CHANGES_REQUESTED** [review]

File: README_AR.rtl.md (hunk 1 above)
Comment (RamanSB) [inline README_AR.rtl.md:1]:
```suggestion
```

**Review by RamanSB — CHANGES_REQUESTED** [review]

File: README_AR.rtl.md (hunk 1 above)
Comment (RamanSB) [reply README_AR.rtl.md:1]: Line removal

**Comment by 0xsisyfos** [comment]
Hello @0xObsidian 

First of all, thank you very much for contributing to the ZK Credo by translating it. We appreciate it greatly, and the community will benefit immensely from your work.

To ensure the quality and consistency of the ZK Credo, we asked a third party to review the translation. We invite you to consider their feedback and make the necessary changes before we accept the PR. Please find the suggestions below. 

Screenshot 2023-11-24 at 11 05 31

Let me know if you have any comments on the suggestions above.

**Comment by 0xObsidian** [comment]
Changes implemented thank you

**Pull request merged**